src/main/resources/liquibase/workbench_changelog/20_1_0.xml

```diff
@@ -0,0 +1,72 @@
+<?xml version="1.1" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+				   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+				   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+		http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.5.xsd">
+
+	<changeSet author="lkovacic" id="v20.1.0-1">
```

Comment:
@lik-83  this will be included in 19_5_0. Please make the changes to the file and changesets

Reply:
Done